Generator/BotBuilderVSIX-V4/BotBuilderTemplatesVSIX.csproj

```diff
@@ -0,0 +1,123 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks like an old-school project file, not a .Net Core / .Net Standard project file. 

Is this the right .Net version?

Reply:
I will double check this.

Reply:
My other VSIX looks similar on the .csproj file. This might be specific for VSIX projects.